### ISSUE TITLE
Focus highlighting does not work with aria-activedescendant 

### ISSUE THREAD

**Issue by JAWS-test** [w3c/aria-practices#1135]
Regarding: 5.6.2 Managing Focus in Composites Using aria-activedescendant

Programs such as Zoomtext or Supernova can highlight the focus. This is important for visually impaired people. If aria-activedescendant is used, however, this does not work because the focus is on the element with aria-activedescendant and not on the element to which aria-activedescendant refers. I.e. in the interest of visually impaired people aria-activedescendant should not be used. It is better if the focused element actually receives the focus. I suggest to include a note about this problem at https://rawgit.com/w3c/aria-practices/apg-1.2/aria-practices.html#kbd_focus_activedescendant and https://w3c.github.io/aria/#aria-activedescendant.

**Comment by JAWS-test** [comment]
Another problem with aria-activedescendant is that the focused entry is often not displayed in the visible area when using the keyboard. The reason is that developers do not pay attention to the note: "Authors SHOULD also ensure that the currently active descendant is visible and in view (or scrolls into view) when focused.".
The problem does not occur without aria-activedescendant, because then the focused element actually gets the focus and is automatically displayed by the browser in the visible area.

**Comment by jnurthen** [comment]
Logging a bug against the screen magnifiers would be a good first step
Standards support issues for VFO can be logged at https://github.com/FreedomScientific/VFO-standards-support/issues/new

**Comment by JAWS-test** [comment]
I didn't see this as a bug of the screen magnifiers, because I assumed that they basically don't evaluate ARIA, but just follow the focus. If screen magnifiers should also take ARIA into account, then I would report the bugs.

**Comment by jnurthen** [comment]
Your second comment implies that they don't follow focus - this is the part I was concentrating on with the bug comment.
For the first comment this sounds like a very reasonable enhancement request that should be made to screen magnifiers.

**Comment by JAWS-test** [comment]
My second comment (visibility of the focused elements) does not refer to the screen magnifiers, but is a general problem for keyboard users.
My first comment refers to screen magnifiers. They follow the focus. The problem with aria-activedescendant is that the real focus of the browser is on a different element than the focus of the application.

**Comment by JAWS-test** [comment]
Nevertheless, it would be worth considering whether the ARIA specification should include a corresponding note, because the bug, if it is one, will certainly not be fixed immediately.

**Comment by jnurthen** [comment]
Not the spec - you are asking for this in aria-practices right?

**Comment by JAWS-test** [comment]
My request related to aria-practices, because there is an extra chapter on the topic in these and because they seem to be the place for such hints. If the ARIA specification also includes such a reference, of course I have nothing against it, but this would be unusual.

**Comment by jnurthen** [comment]
it does not - just wanted to clarify based on your previous comment

**Comment by mcking65** [comment]
Screen magnifiers are assistive technologies, and as such, they are expected to utilize the accessibility tree to provide an accessible experience. So, for any screen mag that is ignoring activedescendant, please raise the bug with its developer.

When the w3c/aria-at project takes on screen mag support, this would certainly be something that it could evaluate.

WRT authors not scrolling correctly, hopefully we have addressed that adequately in the APG. If not, where do you think we should add more guidance?

**Comment by JAWS-test** [comment]
@mcking65 The problem with scrolling is sufficiently described, but unfortunately often ignored. Therefore I close the ticket.

**Comment by JAWS-test** [comment]
The problem occurred on an incorrectly built page. When testing examples on https://rawgit.com/w3c/aria-practice, I found that Zoomtext interprets and displays aria-activedescendant correctly
